src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projectcreation/model/ProjectCreationModel.kt

```diff
@@ -1,30 +1,77 @@
 package org.wycliffeassociates.otter.jvm.app.ui.projectcreation.model
 
+import com.github.thomasnield.rxkotlinfx.observeOnFx
+import io.reactivex.Single
+import io.reactivex.schedulers.Schedulers
+import javafx.beans.property.SimpleBooleanProperty
 import javafx.collections.FXCollections
 import javafx.collections.ObservableList
 import org.wycliffeassociates.otter.common.data.model.Collection
 import org.wycliffeassociates.otter.common.data.model.Language
+import org.wycliffeassociates.otter.common.data.model.ResourceMetadata
+import org.wycliffeassociates.otter.common.data.model.SourceCollection
 import org.wycliffeassociates.otter.jvm.app.ui.chapterpage.model.Project
+import org.wycliffeassociates.otter.jvm.app.ui.inject.Injector
+import org.wycliffeassociates.otter.jvm.persistence.repositories.CollectionRepository
+import org.wycliffeassociates.otter.jvm.persistence.repositories.LanguageRepository
+import org.wycliffeassociates.otter.jvm.persistence.repositories.SourceRepository
+import org.wycliffeassociates.otter.jvm.usecases.CreateProjectUseCase
 import tornadofx.*
 
 class ProjectCreationModel {
+    private val creationUseCase = CreateProjectUseCase(Injector.languageRepo, Injector.sourceRepo, Injector.collectionRepo, Injector.projectRepo)
     var sourceLanguageProperty: Language by property()
-
     var targetLanguageProperty: Language by property()
+    var selectedResource: Collection by property()
+    var selectedAnthology: Collection by property()
+    var selectedBook: Collection by property()
+    var anthologyList: ObservableList<Collection> by property(FXCollections.observableArrayList())
+    var bookList: ObservableList<Collection> by property(FXCollections.observableArrayList())
+
+    val languages: Single<List<Language>>
```

Comment:
Would it be better to turn these into
```
val languages: ObservableList<Language> = FXCollections.observableList(mutableListOf())
...
init {
    creationUseCase
        .getAllLanguages()
        .observeOnFx()
        .subscribe { retrieved ->
            languages.addAll(retrieved)
        }
}

That might keep the viewmodel from having to do the assignment and subscribe to the observable, since I think we only get to retrieve the languages from the use case once at the beginning

Reply:
so much better. thanks